src/python/install.sh

```diff
@@ -218,13 +306,71 @@ add_symlink() {
+    make -j $(nproc)
+    make install_dev
+    )
+    rm -rf /tmp/openssl11
+}
+
+install_gnupg22() {
+    local _prefix=$1
+    yum-builddep -y gnupg2 
+    check_packages bzip2
+    mkdir -p /tmp/gnupg22 && cd /tmp/gnupg22
+    gpg --list-keys
+    gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 249B39D24F25E3B6 04376F3EE0856959 2071B08A33BD3F06 8A861B1C7EFD60D9
+
+    curl -sLfO -C - https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.31.tar.gz.sig && \
+    curl -sLfO -C - https://www.gnupg.org/ftp/gcrypt/libgpg-error/libgpg-error-1.31.tar.gz && \
+    curl -sLfO -C - https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.8.3.tar.gz && \
+    curl -sLfO -C - https://www.gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.8.3.tar.gz.sig && \
+    curl -sLfO -C - https://www.gnupg.org/ftp/gcrypt/libassuan/libassuan-2.5.1.tar.bz2 && \
+    curl -sLfO -C - https://www.gnupg.org/ftp/gcrypt/libassuan/libassuan-2.5.1.tar.bz2.sig && \
+    curl -sLfO -C - https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.3.5.tar.bz2 && \
+    curl -sLfO -C - https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.3.5.tar.bz2.sig && \
+    curl -sLfO -C - https://www.gnupg.org/ftp/gcrypt/npth/npth-1.5.tar.bz2 && \
+    curl -sLfO -C - https://www.gnupg.org/ftp/gcrypt/npth/npth-1.5.tar.bz2.sig && \
+    curl -sLfO -C - https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.2.9.tar.bz2 && \
+    curl -sLfO -C - https://www.gnupg.org/ftp/gcrypt/gnupg/gnupg-2.2.9.tar.bz2.sig && \
+    gpg --verify libgpg-error-1.31.tar.gz.sig && tar -xzf libgpg-error-1.31.tar.gz && \
+    gpg --verify libgcrypt-1.8.3.tar.gz.sig && tar -xzf libgcrypt-1.8.3.tar.gz && \
+    gpg --verify libassuan-2.5.1.tar.bz2.sig && tar -xjf libassuan-2.5.1.tar.bz2 && \
+    gpg --verify libksba-1.3.5.tar.bz2.sig && tar -xjf libksba-1.3.5.tar.bz2 && \
+    gpg --verify npth-1.5.tar.bz2.sig && tar -xjf npth-1.5.tar.bz2 && \
+    gpg --verify gnupg-2.2.9.tar.bz2.sig && tar -xjf gnupg-2.2.9.tar.bz2 && \
+    cd libgpg-error-1.31/ && ./configure --prefix=${_prefix} && make && make install && cd ../ && \
+    cd libgcrypt-1.8.3 && ./configure --prefix=${_prefix} --with-libgpg-error-prefix=${_prefix} && make && make install && cd ../ && \
+    cd libassuan-2.5.1 && ./configure --prefix=${_prefix} --with-libgpg-error-prefix=${_prefix} && make && make install && cd ../ && \
+    cd libksba-1.3.5 && ./configure --prefix=${_prefix} && make && make install && cd ../ && \
+    cd npth-1.5 && ./configure --prefix=${_prefix} && make && make install && cd ../ && \
+    cd gnupg-2.2.9 && LDFLAGS="-Wl,-rpath=${_prefix}/lib" ./configure --prefix=${_prefix} \
+        --with-agent-pgm=${_prefix}/bin/gpg-agent \
+        --with-dirmngr-pgm=${_prefix}/bin/dirmngr \
+        --with-libgpg-error-prefix=${_prefix} \
+        --with-libgcrypt-prefix=${_prefix} \
+        --with-libassuan-prefix=${_prefix} \
+        --with-ksba-prefix=${_prefix} \
+        --with-npth-prefix=${_prefix} \
+        && make && make install && cd ..
```

Comment:
Mmmm... This looks a bit too much for being able to support RHEL for the Python Feature. Can you help me explain your reasoning of adding it? Can we omit most of the bits?

Reply:
Unfortunately, our industry continues to need Centos / RedHat 7, and if we're to support devcontainers on that platform, then we need to be able to install. This "mess" is what it takes to add `dirmgr` from GnuPG to Centos 7.

I could change the logic for downloading the keys to not rely on `dirmgr`, which was, in fact, my first approach. That would mean not adding this large set of things just to install a modern GnuPG onto Centos 7.

Reply:
For centos/redhat 7... I removed the building of gpg2, and replaced it getting gpg keys in a manner that does not need `dirmngr`.

Reply:
Commit [baf1568](https://github.com/devcontainers/features/pull/830/commits/baf1568d80bb6f1eb73dd7e1b774b2fa5f3b662d) addresses this.